extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -428,8 +429,8 @@ export class AmpSelector extends AMP.BaseElement {
       } else {
         this.clearSelection_(el);
       }
-
-      this.fireSelectEvent_(el);
+      // "toggle" action only requires "low" trust.
+      this.fireSelectEvent_(el, ActionTrust.LOW);
```

Comment:
For carousel, we preserve trust rather than taking the min trust. For example, if you trigger a slide change via high trust, the subsequent `slideChange` also gets high trust.

This allows developers to coordinate high trust actions across an asynchronous operation (changing the slide). I think for enabling developers to create consistent experiences, we should allow the same here. 

For example, they may have multiple ways of triggering a selector on a selector. Imagine something higher up on the page, triggering a scrollTo, then changing the selector index as well as a tabbed navigation on top of the selector. If a developer wants to change something as a result of the selector changing, it would be beneficial to listen to the action emitted from the selector component rather than needing to duplicate that logic anywhere that may change the selector. Forcing this to low trust prevents such use cases from doing anything that is high trust.

You can read the trust from the action invocation, and then propagate it here.

Reply:
Good point, done.